internal/discord/commands.go

```diff
@@ -272,7 +272,53 @@ func (d *Discord) Unexile(s *discordgo.Session, i *discordgo.InteractionCreate)
 //	user:		User
 //	reason:		string
 func (d *Discord) Strike(s *discordgo.Session, i *discordgo.InteractionCreate) {
+	optionMap := mapOptions(i)
+	logMsg, _ := d.LogCommand(i.Interaction)
+
+	state := &InteractionState{
+		session:     s,
+		interaction: i,
+		logMsg:      logMsg,
+		isFirst:     true,
+	}
+
+	userToExile := optionMap["user"].UserValue(nil)
```

Comment:
probably should change this to `userToStrike`

Reply:
whoops copy paste mistake. fixed